packages/grpc-js/src/call-stream.ts

```diff
@@ -491,7 +491,7 @@ export class Http2CallStream implements Call {
       try {
         details = decodeURI(metadataMap['grpc-message']);
       } catch (e) {
-        details = metadataMap['grpc-messages'] as string;
+        details = metadataMap['grpc-message'] as string;
```

Comment:
```suggestion
        details = metadataMap['grpc-message'];
```

The `as string` type assertion is only needed because the key didn't match the one in the type check in the surrounding `if` statement, so it can be removed.